docs: scoped token's restricted access does not support standby (#1444)
apify/apify-docs: sources/platform/integrations/programming/api.md

This way you can be sure that Actors won't accidentally—or intentionally—access any data they shouldn't. However, Actors might not function properly if the scope is not sufficient.

> :::caution
> This mode is currently not supported for Actors running in the [Standby mode](/platform/actors/running/standby). You can still send standby requests using a scoped token configured with the restricted access, but we can't guarantee it will always work as expected.
> :::
> 

Reviewer: ```suggestion
:::caution
Restricted access mode is not supported for Actors running in [Standby mode](/platform/actors/running/standby). While you can send standby requests using a scoped token configured with restricted access, functionality is not guaranteed.

```
